framework/test/Volo.Abp.Core.Tests/System/StringExtensions_Tests.cs

```diff
@@ -202,6 +205,12 @@ public void RemovePostFix_Tests()
         [Fact]
         public void RemovePreFix_Tests()
         {
+            //null case
+            (null as string).RemovePreFix("Test").ShouldBeNull();
+
+            //empty case
+            string.Empty.RemovePreFix("Test").ShouldBe(string.Empty);
+
             "Home.Index".RemovePreFix("NotMatchedPostfix").ShouldBe("Home.Index");
```

Comment:
It's a confusing case that call Remove**Pre**Fix with "NotMatched**Post**Fix". Shall we correct the typo?